README.rst

```diff
@@ -147,6 +147,23 @@ Responses as a context manager
         resp = requests.get('http://twitter.com/api/1/foobar')
         resp.status_code == 404
 
+Responses as a pytest fixture
+-----------------------------
+
+.. code-block:: python
+
+    @pytest.yield_fixture
```

Comment:
Shouldn't this be `pytest.fixture`?

Reply:
In newer versions - yes, but that's ok to have it in example, we do not pin to specific pytest version here